manager/controllers/systems_test.go

```diff
@@ -287,6 +287,16 @@ func TestSAPSystemMeta3(t *testing.T) {
 	assert.Equal(t, testMap, output.Meta.Filter)
 }
 
+func TestSAPSystemMeta4(t *testing.T) {
+	url := "/?filter[system_profile][sap_sids][in]=ABC&filter[system_profile][sap_sids][in]=GHI"
```

Comment:
Do we really need to support this? Does it also work for another filters? Can not we we just use:
~~~
filter[system_profile][sap_sids][in]=ABC,GHI
~~~
?

Reply:
I wanted to do this the way you just proposed, but unfortunately FE is already implemented this way `	https://console.stage.redhat.com/api/inventory/v1/hosts?staleness=fresh&staleness=stale_warning&=&registered_with=insights&filter[system_profile][mssql][version]=not_nil&filter[system_profile][sap_sids][]=EM1&filter[system_profile][sap_sids][]=EM2`

I would be more than happy to add support for this but this one gets the job done and we have to move on with other things AFAIK. What do you think about it Pepa?